R/core_functions.R

```diff
@@ -46,8 +46,7 @@ get_tick_den <- function(time, pred, population, developing_population) {
 #'
 #' TODO docs are out of date
 #' @param time First day to get predictor value(s)
-#' @param pred String indicating which predictor, one of: 'temp', 'vpd',
-#'   'host_den' or NA
+#' @param pred A `predictor_spec`
```

Comment:
this input parameter change is why we use `pred$pred` rather than just `pred` throughout this function

Reply:
makes sense